src/website/views/components/steps.pug

```diff
@@ -258,3 +258,127 @@ p.-text
       </li>
     </ul>
 
+h4 Responsiveness
+p.-text
+  | You can use a modifier class with format <code>-{breakpoint}--labels</code> to make responsive the steps component.
```

Comment:
Change text to:
Use `-labels-{breakpoint}--hide` to hide step component labels on specific breakpoints or `-labels--hide` to hide on all breakpoints. Breakpoints supported are `sm`, `md`, `lg`, and `xl`. Labels are shown on all breakpoints by default.

Reply:
done